inspire_mitmproxy/errors.py

```diff
@@ -37,20 +35,27 @@ def __init__(self, request: dict) -> None:
 
```

Comment:
^ 501 Not Implemented  here too (or something it the 5xx range).

Reply:
Done